dotnetv3/dynamodb/low-level-api/LowLevelTableExample/LowLevelTableExample.cs

```diff
@@ -80,18 +80,12 @@ private static async Task CreateExampleTable()
                     KeyType = KeyType.RANGE //Sort key
                 }
             },
-            ProvisionedThroughput = new ProvisionedThroughput
-            {
-                ReadCapacityUnits = 5,
-                WriteCapacityUnits = 6
-            },
-            TableName = ExampleTableName
+            TableName = ExampleTableName,
+            BillingMode = BillingMode.PAY_PER_REQUEST,
```

Comment:
This example works. Its old style where the program runs without a user clicking Enter. Not a requirement, but it is nice to have a pause so user can understand what is happening.